floyd_warshall_algorithm/CPP/floyd_warshall_algorithm.cpp

```diff
@@ -0,0 +1,67 @@
+// Floyd-Warshall Algorithm: all pair shortest path
+#include <iostream>
+#include <climits>
+using namespace std;
+
+#define INF 999999999
+
+int main() {
+	freopen("input.txt", "r", stdin);
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.cpp`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpuezpmrt3/floyd_warshall_algorithm/CPP/floyd_warshall_algorithm.cpp
+++ b/tmp/tmpuezpmrt3/floyd_warshall_algorithm/CPP/floyd_warshall_algorithm.cpp
@@ -10,7 +10,7 @@
 	int i, j, k, n, m;
 	cin >> n >> m;	// number of vertices and edges respectively
 	int graph[n+1][n+1];
-	
+
 	for(i = 1; i <= n; i++) {
 		for(j = 1; j <= n; j++)
 			if(i == j)	graph[i][j] = 0;	// Shortest distance between (i,i) is 0
```